controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -763,6 +763,16 @@ private File mkdir(final File dir) {
         return dir;
     }
 
+    private static boolean forcedMove(Path from, Path to) {
+        try {
+            Files.move(from, to);
```

Comment:
OK, will do. This is only used in cases where 'from' and 'to' are dirs, and with REPLACE_EXISTING Files.move will fail if 'to' exists and is non-empty. But without REPLACE_EXISTING it will fail if 'to' exists, regardless of whether it's empty, so adding the option may help.